arrow-fx-test/src/main/kotlin/arrow/test/laws/ConcurrentLaws.kt

```diff
@@ -624,18 +625,18 @@ object ConcurrentLaws {
       }.equalUnderTheLaw(just(a + b + c), EQ)
     }
 
-  fun <F> Concurrent<F>.parMapStartsAllAtSameTime(EQ: Eq<Kind<F, List<Long>>>) {
-    val order = mutableListOf<Long>()
+  fun <F> Concurrent<F>.parMapStartsAllAtSameTime(EQ: Eq<Kind<F, Any>>) {
```

Comment:
😱 `Eq<Kind<F, Any>>` should be `Eq<Kind<F, Tuple6<Int, Int, Int, Int, Int, Int>>>`